olive/common/ort_inference.py

```diff
@@ -52,6 +56,21 @@ def get_ort_inference_session(
         from onnxruntime_extensions import get_library_path
 
         sess_options.register_custom_ops_library(get_library_path())
+    if external_initializers:
+        from onnxruntime import OrtValue
+
+        # convert external initializers to OrtValue
+        initializer_names = []
+        initializer_values = []
+        for name, value in external_initializers.items():
+            initializer_names.append(name)
+            initializer_values.append(OrtValue.ortvalue_from_numpy(value))
+
+        # add external initializers to the session
+        sess_options.add_external_initializers(initializer_names, initializer_values)
+
+        # delete external initializers to free up memory
+        del external_initializers
```

Comment:
the del should be in callers?

Reply:
Oh, I forgot about this. Will remove it.

Reply:
Done.